### PR TITLE
Add Check-in Events funcationality for Hackers

### LINKED ISSUE #677: Add check-in for events during the hackathon
We need to track event attendance of users during the hackathon. Since we will give out swag to those who attend events, we would need a way to know how many events people attend, and what the addresses of those who attend many events.

One way we were thinking is adding a check-in button on Vaken main page (it will check in for different events depending on times of the event) or seperate pages with unique URL's which we will send out during the event, and clicking the button will add an event to eventsAttended field for a user. 

If we have eventsAttended field populated after the hackathon, we will fetch top event attenders and send them swag to the address indicated during the registration process.

### PR DESCRIPTION
**Pull request by BaibhaVatsa**
Flow: Organizer has to pull events from calendar inside `manage events` once so that the events are created in the db. Then, the `check-in` shows at any time the ongoing events which people can check-in to.
ongoingevent

Closes #677

### CODE REVIEW

File: src/server/resolvers/index.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { UserInputError, AuthenticationError } from 'apollo-server-express';
 import { ObjectID } from 'mongodb';
+import { async } from 'node-ical';
```
Comment (gfting) [inline src/server/resolvers/index.ts:3]: @BaibhaVatsa what is this doing here? I'm not quite familiar with this, and wouldn't this potentially cause some naming conflicts?

**Review by gfting — COMMENTED** [review]
Looks good! Just one question.

File: src/server/resolvers/index.ts (hunk 1 above)
Comment (BaibhaVatsa) [reply src/server/resolvers/index.ts:3]: Wait lmao. I don't think I added this manually.

**Comment by gfting** [comment]
@BaibhaVatsa there are also some TS compilation errors, could you take a look?

**Comment by BaibhaVatsa** [comment]
@gfting those should be fixed now

**Pull request merged**